frontend/pages/index.tsx

```diff
@@ -1,11 +1,14 @@
 import React from "react";
 
+import { FeaturesList } from "../components/FeaturesList";
 import { HeroSection } from "../components/HeroSection";
+import styles from "../styles/Home.module.css";
 
-export default function Home(): JSX.Element {
+export default function Home(): React.ReactNode {
   return (
-    <>
+    <div className={styles.container}>
```

Comment:
There's no need for this extra div! + styles We have our container wrapper implemented in `frontend/pages/_app.tsx:25`